### PR TITLE
Remove entrypoint from Dockerfile

### PR DESCRIPTION
**Pull request by rolandpoulter**
This makes it easier to debug docker images and containers.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.bintray.json
+.dockerignore
+.git
+.gitignore
+.hound.yml
+.jshintrc
+.travis.yml
+dhcpd.conf
```
Comment (benbp) [inline .dockerignore]: Do we not want to track changes to this?

**Comment by benbp** [comment]
:+1: but minor question about tracking dhcpd.conf

File: .dockerignore (hunk 1 above)
Comment (rolandpoulter) [inline .dockerignore]: No, it doesn't use that version of dhcp.conf. That is provided here: https://github.com/RackHD/RackHD/pull/86/files#diff-007d36abd8a3deadeee5749bf1f84c98

I'm trying to limit the docker containers to the the bare essential files. When you `ls` the directory in the container you should only get:
```
usxxgardij1m1:docker poultr2$ docker run -it rackhd/on-dhcp-proxy /bin/sh
/RackHD/on-dhcp-proxy # ls
debian        index.js      lib           node_modules  package.json
/RackHD/on-dhcp-proxy # 
```

Comment (rolandpoulter) [inline .dockerignore]: Oh, lol, should take out `debian/` as well.

**Comment by benbp** [comment]
Oh, duh, because we run isc-dhcp-server in on-taskgraph right now.

**Comment by VulpesArtificem** [comment]
:+1:

**Pull request merged**